x-pack/test/reporting_api_integration/reporting_and_security/usage/api_counters.ts

```diff
@@ -43,7 +44,7 @@ export default function ({ getService }: FtrProviderContext) {
       enum paths {
         LIST = '/api/reporting/jobs/list',
         COUNT = '/api/reporting/jobs/count',
-        INFO = '/api/reporting/jobs/info/{docId}',
+        INFO = '/api/reporting/jobs/info/kraz0qle154g0763b569zz83',
```

Comment:
These tests needed maintenance, as the usage counters are now only counting valid requests: not 404s or unauthorized.